openshift-hack/e2e/annotate/annotate.go

```diff
@@ -231,28 +231,14 @@ func (r *ginkgoTestRenamer) generateRename(name string, node types.TestSpec) {
 			newLabels += " [Suite:openshift/conformance/parallel]"
 		}
 	}
-	codeLocations := node.CodeLocations()
-	if isGoModulePath(codeLocations[len(codeLocations)-1].FileName, "k8s.io/kubernetes", "test/e2e") {
-		newLabels += " [Suite:k8s]"
-	}
+	newLabels += " [Suite:k8s]"
```

Comment:
Interesting, I use a different path as well, but I don't have this issue. Also, `k8s.io/kubernetes` is a module, not a path. The patch here would be `test/e2e`.

@smg247 is it worth spending time on this? Are we going to drop annotations soon?